modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSViews.tsx

```diff
@@ -219,11 +219,71 @@ const FDSViews = ({
 		},
 	];
 
+	const onDeleteClick = ({
+		itemData,
+		loadData,
+	}: {
+		itemData: any;
+		loadData: Function;
+	}) => {
+		openModal({
+			bodyHTML: Liferay.Language.get(
+				'deleting-a-dataset-view-is-an-action-that-cannot-be-reversed'
+			),
+			buttons: [
+				{
+					autoFocus: true,
+					displayType: 'secondary',
+					label: Liferay.Language.get('cancel'),
+					type: 'cancel',
+				},
+				{
+					displayType: 'danger',
+					label: Liferay.Language.get('delete'),
+					onClick: ({processClose}: {processClose: Function}) => {
+						processClose();
+
+						fetch(`${fdsViewsAPIURL}/${itemData.id}`, {
+							method: itemData.actions.delete.method,
```

Comment:
We should probably hardcode `DELETE` here, since we are using endpoint directly.